taxonomies/ingredients.txt

```diff
@@ -487,7 +487,7 @@ el:Γαλακτωματοποιητής
 es:Emulgente, Emulsificante
 et:Emulgaator
 fi:Emulgointiaine
-fr:Émulsifiant
+fr:Émulsifiant, émulsiﬁant
```

Comment:
Misspelling, should not be added.

Reply:
Is this a misspelling? They used a ﬁ, which is a single character.

Reply:
the "fi" character is not used in French, this one probably comes from an OCR engine